openstack/autoscaling/v1/groups_hcs/delete.go

```diff
@@ -0,0 +1,8 @@
+package groups_hcs
```

Comment:
hcs - what is it?

Reply:
I don't know but https://github.com/opentelekomcloud/gophertelekomcloud/tree/devel/openstack/autoscaling/v1/groups_hcs

Reply:
This is an old version of the code that was deprecated, so I removed it.